docs/reST/ref/font.rst

```diff
@@ -266,6 +281,10 @@ solves no longer exists, it will likely be removed in the future.
       to use for the text background. If no background is passed the area
       outside the text will be transparent.
 
+      Starting in `2.1.4`, there is a `wraplength` argument which describes the
```

Comment:
I don't know if "Starting in `2.1.4`," is needed as it's been stated at the bottom already.